src/main/webapp/app/exercises/shared/exercise-update-warning/exercise-update-warning.component.html

```diff
@@ -0,0 +1,36 @@
+<div style="padding: 20px">
+    <h2><span jhiTranslate="artemisApp.exercise.update.warning.warning"></span></h2>
+    <div style="padding: 10px">
+        <h4><span jhiTranslate="artemisApp.exercise.update.warning.problems"></span></h4>
+        <ul>
+            <li *ngIf="instructionDeleted" style="padding-top: 5px">
+                <span jhiTranslate="artemisApp.exercise.update.warning.instructionDeleted"></span>
+                <ul style="padding-left: 15px">
+                    <li style="margin-bottom: -10px">
```

Comment:
New css classes can be added so you can change both instruction deleted and scoring changed at the same time